Gemfile

```
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 
 # You may use http://rbenv.org/ or https://rvm.io/ to install and use this version
-ruby '>= 3.0.0'
+ruby '3.1.4'
```

Why pin ruby to specific patch version? 
```suggestion
ruby '>= 3.1.4'
```